src/components/Nav.js

```diff
@@ -17,9 +17,9 @@ function LeftSide({isLoggedIn, user, handleLogIn, handleLogOut}) {
   };
 
   return (
-    <FloatLeftMobileNav>
-      <Link style={{verticalAlign: "middle"}} to="/">
-        <img alt="open sauced" src={logo} />
+    <FloatLeftMobileNav >
+      <Link to="/" style={{display: "inline-block", verticalAlign: "middle"}}>
+        <img alt="open sauced" src={logo} style={{display: "inline-block", verticalAlign: "middle", marginBottom: 6}} />
```

Comment:
@JapneetSingh5 Moving to **styled-components** would be awesome!

Reply:
Oh okay. I don’t have much experience with styled-components, will find out how to use them.

Reply:
Oh right, they seem familiar. Do i create a new component named, say NavLogo and put all the styles in there?

Reply:
@JapneetSingh5 Instead of new comp! We can create a class called `.logo` inside `FloatLeftMobileNav`!

WDYT?

Reply:
Made a change, have a look.